README.md

```diff
@@ -1,2 +1,38 @@
-# Bright Futures Academy
-http://brightfuturesatl.org/the-program/academy/
+# Bright Futures Academy Parent Portal
+This is a web app built to manage payroll at [Bright Futures Academy](
+http://brightfuturesatl.org/the-program/academy/) in Atlanta.
+It provides information for prospective students, and enrolled families
+can sign in to manage their students' tuition and other payments. 
+
+## Getting Started
+### Prerequisites
+You'll need [npm](https://nodejs.org/en/) to run this project.
+
+### Installing
+First, install the packages for both the client and server.
+```
+npm run postinstall
+```
+
+## Running the app
+You can start a development server like this:
+```
+npm run server
+```
+Or you can run the full app like this:
+```
+npm run dev
+```
+
+## Contributing
+All issues are kept under the `Projects` tab. If you find anything you'd like to
+add or see fixed, add it there. 
+
+If you'd like to contribute, create a new branch and make changes as you'd like. 
+Pull requests should to be reviewed by at least one active contributor. 
+Please include screenshots with any frontend changes.
```

Comment:
Add in:

To make a pull request:
1) Make a branch off of `master` by doing `git checkout -b <branch name>`
2) Make your changes and then do `git add .`, `git commit -m "I made xyz changes"`, `git push --set-upstream origin <branch name>`
3) Go back to github and you'll see an option to open up a PR.